<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeAssignmentStrip</class>
 <widget class="QFrame" name="timeAssignmentStrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>73</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel { color: black; } #timeAssignmentStrip { background-color: qlineargradient( x1:0 y1:0, x2:0 y2:1, stop:0 #7FFFFF, stop:1 #6FBFBF); }</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::WinPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" stretch="0,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" stretch="0,1,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="labelTime">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { background-color: qlineargradient( x1:0 y1:0, x2:0 y2:1, stop:0 #EEEEEE, stop:1 #BBBBBB); }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string notr="true">9:99</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProject">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,30);</string>
       </property>
       <property name="text">
        <string notr="true">PROJECT 1
12345</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelId">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,30);</string>
       </property>
       <property name="text">
        <string notr="true">0123456</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="labelSubproject">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,30);</string>
       </property>
       <property name="text">
        <string notr="true">Subproject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWorkpackage">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,30);</string>
       </property>
       <property name="text">
        <string notr="true">Workpackage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelText">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,30);</string>
       </property>
       <property name="text">
        <string notr="true">Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
